extensions/wikia/AuthModal/js/AuthModal.js

```diff
@@ -52,49 +35,42 @@ define('AuthModal', ['jquery', 'wikia.window'], function ($, window) {
 		return track;
 	}
 
-	function onKeyUp (event) {
-		if (event.keyCode === 27) {
-			close();
-		}
-	}
-
 	function close (event) {
 		if (event) {
 			event.preventDefault();
 		}
-
-		if (modal) {
+		if (authPopUp) {
 			track({
```

Comment:
Please make sure that tracking is called only once when you call this method. See: `onbeforeunload`